main.nf

```diff
@@ -1,14 +1,179 @@
 #!/usr/bin/env nextflow
 
 nextflow.enable.dsl=2
+idList = []     // Global variable to save IDs from samplesheet
 
 include { CALL_STAPHYLOCOCCUS_AUREUS        } from './workflows/staphylococcus_aureus.nf'
 include { CALL_MYCOBACTERIUM_TUBERCULOSIS   } from './workflows/mycobacterium_tuberculosis.nf'
 
+<<<<<<< HEAD
+
+
+// Function for platform and paired-end or single-end
+def get_meta(LinkedHashMap row) {
```

Comment:
This has been moved to [methods/get_meta.nf](https://github.com/genomic-medicine-sweden/JASEN/blob/master/methods/get_meta.nf). Remove/copy changes there:)